### PR TITLE
KOGITO-9451 Reorganize SonataFlow Tooling Guides TOC

### PR DESCRIPTION
**Pull request by paulovmr**
**JIRA:** [*KOGITO-9451*](https://issues.redhat.com/browse/KOGITO-9451)

**Description:** In this PR, we only review the Tooling guides TOC with the content we already have and left the navigation page ready for the new content. Please see [KOGITO-9451](https://issues.redhat.com/browse/KOGITO-9451) for more information.

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] The nav.adoc file has a link to this guide in the proper category
- [ ] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

**Review by ricardozanini — APPROVED** [review]

**Review by kaldesai — APPROVED** [review]
Your changes look good to me. Thank you!

File: serverlessworkflow/modules/ROOT/nav.adoc
Comment (MarianMacik) [inline serverlessworkflow/modules/ROOT/nav.adoc:56]: These issues are still Open.

File: serverlessworkflow/modules/ROOT/nav.adoc
Comment (MarianMacik) [inline serverlessworkflow/modules/ROOT/nav.adoc:29]: Maybe better to put it just after the VS Code extension?

File: serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc
Comment (MarianMacik) [inline serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc:1]: Maybe
```suggestion
= Chrome extension for Serverless Workflow editor on GitHub
```
?
But I am not sure.

File: serverlessworkflow/modules/ROOT/nav.adoc
Comment (paulovmr) [reply serverlessworkflow/modules/ROOT/nav.adoc]: Done.

File: serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc
Comment (paulovmr) [reply serverlessworkflow/modules/ROOT/pages/tooling/serverless-workflow-editor/swf-editor-chrome-extension.adoc]: Yes, that is right. Changed.

**Review by MarianMacik — APPROVED** [review]
Looks good now, thanks!

**Review by ederign — APPROVED** [review]

**Comment by ederign** [comment]
@ricardozanini can we merge this? This blocks other tooling docs

**Review by domhanak — APPROVED** [review]

**Comment by ricardozanini** [comment]
> @ricardozanini can we merge this? This blocks other tooling docs

I was waiting for QE approval. I'll merge it now.

**Pull request merged**
